unit_tests/trilinos_interfaces/testKokkosInit/testKokkosInit.f90

```diff
@@ -10,10 +10,13 @@ PROGRAM test_kokkos_init
   USE ISO_C_BINDING
   USE trilinos_interfaces
   IMPLICIT NONE
-
+#ifdef FUTILITY_HAVE_Kokkos
   !Initializing Kokkos.
   CALL InitializeKokkos(4,-1)
+  WRITE(*,*) "Kokkos initialization successful."
   !Finalizing Kokkos.
   CALL FinalizeKokkos()
+  WRITE(*,*) "Kokkos finalization successful."
```

Comment:
Do we want these 2 write statements on master?

Reply:
Didn't realize I was looking at a test, so I actually don't really care about these.  Let's just go ahead and push this in.